configuration/jsr223.md

```diff
@@ -54,6 +60,51 @@ automationManager.addRule(sRule);
 
 :::
 
+::: tab JS&nbsp;Scripting
+
+```js
+const { automationManager } = require('@runtime/RuleSupport');
+const Configuration = Java.type('org.openhab.core.config.core.Configuration');
+const SimpleRule = Java.extend(Java.type('org.openhab.core.automation.module.script.rulesupport.shared.simple.SimpleRule'));
```

Comment:
Is there a good reason to show using the low level API's here ?  Should we just show openhab-js examples since thats what ships with it, and new users will likely end up here and be confused ?

Reply:
Because this page is explicitly documenting the low level API and it's confusing to show an example that doesn't use the low level API on this page. If using just openhab-js there would be nothing on this page explaining how that example works and conversely there is nothing that the example would be demonstrating that is documented on this page.